framework/Runners/__init__.py

```diff
@@ -25,7 +25,7 @@
 ## 'from Runners.Runner import Runner' outside
 ## of this submodule
 from .Runner import Runner
-from .ExternalRunner import ExternalRunner
+# from .ExternalRunner import ExternalRunner
```

Comment:
Yes, this vestigial piece should be removed from this file so as not to promote confusion later on. Good catch!